cpp/test/sg/umap_parametrizable_test.cu

```diff
@@ -258,6 +258,8 @@ class UMAPParametrizableTest : public ::testing::Test {
       return;
     }
 
+#if CUDART_VERSION >= 11020
```

Comment:
Could you please leave a comment on what's missing from the previous cuda version?